common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -15,7 +15,7 @@
 from ..helpers import EventsTestMixin
 
 
-class ProblemsTest(UniqueCourseTest):
+class ProblemsTest(UniqueCourseTest, EventsTestMixin):
```

Comment:
Actually, I'm wondering if you could not-make this improvement at the moment. The EventsTestMixin is problematic for multiprocess tests right now, so if we keep its footprint smaller we'll have more options.